### PR TITLE
Added tasks command and integration test

### PR DESCRIPTION
**Pull request by zhanggbj**
Hi @maximilien,
This is bmp tasks command and its integration test. Thanks for your review.

[#115478103]

### CODE REVIEW

File: cmds/bmp/tasks_command.go [hunk 1]
```diff
@@ -51,5 +55,41 @@ func (cmd tasksCommand) Validate() (bool, error) {
 
 func (cmd tasksCommand) Execute(args []string) (int, error) {
 	cmd.printer.Printf("Executing %s comamnd: args: %#v, options: %#v", cmd.Name(), args, cmd.options)
+
+	var latest uint
+	latest = 50
```
Comment (maximilien) [inline cmds/bmp/tasks_command.go]: I wonder if we can pass this as a optional parameter to the command and default to 50?

File: cmds/bmp/tasks_command.go [hunk 2]
```diff
@@ -51,5 +55,41 @@ func (cmd tasksCommand) Validate() (bool, error) {
...
+	table.SetHeader([]string{"Task ID", "Status", "Description", "Start", "End"})
+
+	length := len(tasksResponse.Data)
+	content := make([][]string, length)
+	for i, task := range tasksResponse.Data {
+		content[i] = []string{
+			strconv.Itoa(task.Id),
+			task.Description,
+			task.Status,
+			task.Start_time,
```
Comment (maximilien) [inline cmds/bmp/tasks_command.go]: Why is this named `Start_time` vs `StartTime`? Motivate this choice, please. Same as `End_time`

File: cmds/bmp/tasks_command_test.go [hunk 3]
```diff
@@ -72,10 +74,57 @@ var _ = Describe("BMP: tasks command", func() {
...
+				rc, err := cmd.Execute([]string{"bmp", "tasks"})
+				Expect(rc).To(Equal(0))
+				Expect(err).ToNot(HaveOccurred())
+			})
+
+			It("executes a good TasksCommand with specifying latest", func() {
+				cmd = bmp.NewTasksCommand(options, fakeBmpClient)
+
+				rc, err := cmd.Execute([]string{"bmp", "tasks", "--latest=1"})
+				Expect(rc).To(Equal(0))
```
Comment (maximilien) [inline cmds/bmp/tasks_command_test.go:97]: We need to verify that one task or at a minimum something was written in the output. Maybe you can make sure output contains the correct total? Same as above.

**Comment by maximilien** [comment]
Grace, please address comments and I will merge. Also, make sure to always include the tracker story number in all commits. I added to the PR message. You also need to start the story and finish and deliver when done. 

[#115478103]

**Comment by zhanggbj** [comment]
Hi dr.max,
Thanks for your reminder. I will address these comments today. And I forgot to mention that this PR is still  WIP. This commit only implemented part of the story #115478103 which include two command bmp tasks and bmp task {id}. I'm still working on the bmp task {id}

**Comment by maximilien** [comment]
Grace, maybe it's best to work on this and next PR and meet with me tomorrow instead?

**Comment by zhanggbj** [comment]
OK. I'll focus on this PR and finish it firstly. And I have just rescheduled the meeting. Thank you.

File: cmds/bmp/tasks_command.go (hunk 2 above)
Comment (zhanggbj) [inline cmds/bmp/tasks_command.go]: Done

File: cmds/bmp/tasks_command_test.go (hunk 3 above)
Comment (zhanggbj) [inline cmds/bmp/tasks_command_test.go:97]: How about we verify the output in the tasks integration test? The length of cmd tasks response is also verified in the bmp client UT.

File: cmds/bmp/tasks_command.go (hunk 1 above)
Comment (zhanggbj) [inline cmds/bmp/tasks_command.go]: As far as I know, it seems golang doesn't have optional parameter for functions.
http://stackoverflow.com/questions/2032149/optional-parameters

File: cmds/bmp/tasks_command_test.go (hunk 3 above)
Comment (maximilien) [inline cmds/bmp/tasks_command_test.go:97]: Cool 👍

**Comment by maximilien** [comment]
Let me know when you want me to review again?

File: cmds/bmp/tasks_command.go (hunk 1 above)
Comment (zhanggbj) [inline cmds/bmp/tasks_command.go]: Fixed this. I misunderstood last time.

**Comment by zhanggbj** [comment]
Hi dr.max @maximilien ,

Please help to review again. Thanks

**Comment by maximilien** [comment]
OK cool merging. We do need a few more tests but we can add them latter as chores.

**Pull request merged**